source/MaterialXGenOsl/ArnoldShaderGenerator.cpp

```diff
@@ -12,10 +12,6 @@ ArnoldShaderGenerator::ArnoldShaderGenerator()
     const StringSet restrictedNames = { "translucent", "empirical_bssrdf", "randomwalk_bssrdf", "volume_absorption",
                                         "volume_emission", "volume_henyey_greenstein", "volume_matte" };
     _syntax->registerRestrictedNames(restrictedNames);
-
-    // Remap struct shader outputs due issues with Arnold renderer. 
-    // Remove this when this issue has been addressed.
-    _remapShaderOutput = true;
```

Comment:
Removing output type remapping since it seems to be fixed on OSL side now. The OSL compiler are not crashing any more.